### ISSUE TITLE
Admin Login Panel Customization

### ISSUE THREAD

**Issue by ryanhmaas** [Gottwik/Enduro#181]
I looked around but didn't see anything in the docs regarding the admin login panel - is it possible to customize (or override) the login UI, including the logo? Apologies if I missed it.

**Comment by rnlambert7** [comment]
Yes. There are two main ways depending on how you wish to customize it. The admin panel itself is an Enduro app so you can head to its [github](https://github.com/Gottwik/Enduro_admin) and check it out. There are instructions on how to npm link it so that while running it separately your current project uses it.

The other way is to use the admin extensions functionality. Inside the example app you can see an example of a js file inside `assets/admin_extensions` that gets called on the admin. This would be useful if you just need to override something or add something small.
